ParkingLotApi/Service/ParkingLotApiService.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using ParkingLotApi.Data_Entity;
+using ParkingLotApi.DTO;
+using ParkingLotApi.Repository;
+
+namespace ParkingLotApi.Service
+{
+    public class ParkingLotApiService
+    {
+        private readonly ParkingLotContext parkingLotDbContext;
+        private int pageSize = 15;
+
+        public ParkingLotApiService(ParkingLotContext parkingLotDbContext)
+        {
+            this.parkingLotDbContext = parkingLotDbContext;
+        }
+
+        public async Task<int> AddParkingLotAsnyc(ParkinglotDTO parkinglotDto)
+        {
+            if (await parkingLotDbContext.Parkinglots.FirstOrDefaultAsync(parkinglot => parkinglot.Name == parkinglotDto.Name) != null)
+            {
+                return -1;
```

Comment:
set a specific definition for -1 seems better

Reply:
agreed